backend/app/tests/test_app_timezone.py

```
@@ -0,0 +1,98 @@
+
+    berlin = Settings(APP_TIMEZONE="Europe/Berlin")
+    monkeypatch.setattr(mail_sources_api, "get_settings", lambda: berlin)
+
+    source = SimpleNamespace(
+        id=1,
+        name="inbox",
+        method="IMAP",
+        server="imap.example.com",
+        port=993,
+        username="user",
+        password="secret",
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)